lib/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -140,14 +142,21 @@ public String getSigningKeyId() {
         return ssmProvider.get(getEnvironmentVariable(SIGNING_KEY_ID_PARAM));
     }
 
-    public CredentialIssuerConfig getCredentialIssuer(String credentialIssuerId) {
+    public CredentialIssuerConfig getCredentialIssuerActiveConnectionConfig(
+            String credentialIssuerId) {
         Map<String, String> result =
                 getSsmParameters(
                         String.format(
-                                "%s/%s",
+                                "%s/%s/connections/%s",
                                 getEnvironmentVariable(CREDENTIAL_ISSUERS_CONFIG_PARAM_PREFIX),
-                                credentialIssuerId));
-        return new ObjectMapper().convertValue(result, CredentialIssuerConfig.class);
+                                credentialIssuerId,
+                                getActiveConnection(credentialIssuerId)));
+
+        CredentialIssuerConfig credentialIssuerConfig =
```

Comment:
Didnt we want to give rid of this and make changes in the code to reference the Id other ways?